packages/contracts/contracts/default-apps/ETHBucket.sol

```diff
@@ -8,15 +8,16 @@ import "../libs/LibOutcome.sol";
 contract ETHBucket is CounterfactualApp {
 
   struct AppState {
-    LibOutcome.CoinTransfer[] transfers;
+    address[] tokens;
+    LibOutcome.CoinTransfer[][] balances;
```

Comment:
should add a comment specifying what the two indices into `balances` are (I assume they are token address and then recipient)